### ISSUE TITLE
Sort imports using package first, then items

### ISSUE THREAD

**Issue by supermonkeybrainz** [eslint/eslint#13900]
The reasoning for this rule change is the frustration of imports constantly being shifted around when changing what gets imported from a given package.  Also, it is awkward that imports from related packages get separated in the import list.  This should minimize the shifting of imports and keep related items together by default (consequently simplifying pull requests).

**What rule do you want to change?**
[sort-imports](https://eslint.org/docs/rules/sort-imports)
**Does this change cause the rule to produce more or fewer warnings?**
More
**How will the change be implemented? (New option, new default behavior, etc.)?**
New option, and ideally new default behavior
**Please provide some example code that this change will affect:**

```js
// Correct
import AnotherThing from 'Lib1/Package1'
import Thing from 'Lib1/Package2'
import Foo from 'Lib2/Package1'
import Bar from 'Lib2/Package2'

// Incorrect
import AnotherThing from 'Lib1/Package1'
import Bar from 'Lib2/Package2'
import Foo from 'Lib2/Package1'
import Thing from 'Lib1/Package2'
```

**What does the rule currently do for this code?**
Sorts imports by the items being imported ONLY
**What will the rule do after it's changed?**
Sorts imports first by package, then by items being imported
**Are you willing to submit a pull request to implement this change?**
Yes

**Comment by mdjermanovic** [comment]
Hi @supermonkeybrainz, thanks for the issue!

This rule is designed to sort declarations by syntax and import specifiers, and it isn't likely that this logic will be changed.

For sorting by modules you can use [import/order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) rule with the [alphabetize](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#alphabetize-order-ascdescignore-caseinsensitive-truefalse) option.

**Comment by supermonkeybrainz** [comment]
Didn't know about that one, perfect. :)
